docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -224,6 +224,11 @@ Types of instruments currently available:
   <xref:System.Diagnostics.Metrics.Histogram%601.Record%2A> to record these measurements during the collection tool's update interval: 1,5,2,3,10,9,7,4,6,8. A collection tool
   might report that the 50th, 90th, and 95th percentiles of these measurements are 5, 9, and 9 respectively.
 
+  > [!NOTE]
+  > For details about how to set the recommended bucket boundaries when creating
```

Comment:
Do we need to tell somewhere `InstrumentAdvice<T>` can be used in the future with more instruments as the scenario permit?

Reply:
Up to you. I feel like we don't need to mention it now. When we have more advice features we can make doc updates to show them?